frontend/components/templates/layouts/index.tsx

```diff
@@ -17,6 +17,7 @@ const Layout = ({ children }: LayoutProps) => {
         '/teams/[slug]/manage',
         '/teams/[slug]/question/[question-slug]',
         '/teams/[slug]/question/[question-slug]/edit',
+        '/roles',
```

Comment:
```suggestion
        '/manage/roles',
```

All admin pages will fall under the _manage_ submenu in the sidebar. Please also relocate _roles_ in the pages folder.